main.js

```diff
@@ -0,0 +1,86 @@
+let turn = "X"
+let turnCounter = 0;
+
+const cells = Array.from(document.getElementsByClassName("cell"))
+const board = document.getElementById("board")
+const clearBtn = document.getElementById("clear-btn")
+const turnHeader = document.getElementById("turn-header")
+
+function boardClicked(event) {
+    const cell = event.target
+
+    console.log(cell.id + " clicked")
+
+    if (cell.className != "cell" || cell.innerText !== "") {
```

Comment:
`cell.classList.has('cell')` will allow usage of other classes without breaking the logic